app/components/add-layer/style.scss

```diff
@@ -46,3 +55,60 @@
 .add-layer .btn-block:hover {
     background: rgba(255, 255, 255, 0.60);
 }
+
+
+
+.layer-links {
+  max-width: (($size * 24) + (($space * $size)* 2)) * 3;
+  display: inline-flex;
+  flex-wrap: wrap;
+}
+
+.layer-link {
+  width: 3em;
+  height: 3em;
+  font-size: $size;
+  text-decoration: none;
+  background: $gray;
+  color: $text;
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+  margin: $space * 1em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all .4s ease-out;
+  transform: translateZ(0px); // tip from @trajektorijus, thank you!
+
+  &:hover, 
+  &:focus, 
+  &:active {
+    background: $accent;
+    color: #fff;
+    z-index: 1;
+    box-shadow: 0 0 ($size / 2) 0 rgba(#000, .5);
+    transform: scale($zoom);
+    transition: .3s;
+
+    &:after {
+      bottom: 1em;
+      transition: .3s;
+    }
+
+  }
+
+  &:after {
+    content: attr(title);
+    width: 100%;
+    text-align: center;
+    left: 0;
+    color:white;
+    bottom: ($size * -3);
+    font-size: .35em;
+    position: absolute;
+    transition: .3s;
+    font-family: 'Alegreya Sans SC', sans-serif;
```

Comment:
This looks like a new font but where is it loading? 

Also can we not use a new font? Are these icon fonts?